### DISCUSSION TITLE
Can data be imported from OTHER ERP software for example QuickBooks?

### DISCUSSION THREAD

**Discussion by ACitizen25** [bigcapitalhq/bigcapital#297] [Q&A]
Can data be imported from OTHER ERP software for example QuickBooks?

What data can be imported into this from outside of the software.

Also can data be export too?

**Comment by c42705** [comment]
I
I have seen an import option on several screens. It takes CSV files. And mapping probably. Try it

**Comment by jesus-ministries** [comment]
I'm just starting to play around with BC .. I generated a customer list and you can go to the customer import screen in BC and it will pull in the XLSX files directly (in lieu of any CSV's) but the mapping is a little messy at the moment.  For instance, BC wants individual values for someone's address/contact info but Quickbooks will provide a field called "Bill to 3" which has data in the format of "City, State, Zip-code" for a contact in the USA.  So, if you want your data handled properly in BC, some massaging is necessary to get the customer data split into separate fields in the spreadsheet before importing to BC.  This is as far as I've gotten so far -- I figured I'd start with our customer list.  I can't speak for other areas of BC such as chart of accounts, transactions or anything else as I've not yet tried.
